packages/playground/src/components/select_solution_flavor.vue

```diff
@@ -66,11 +66,11 @@ import type { solutionFlavor } from "../types";
 type Package = PropType<solutionFlavor>;
 
 const props = defineProps({
-  minimum: { type: Object as Package, default: () => ({ cpu: 1, memory: 1024 * 2, disk: 15 }) },
-  standard: { type: Object as Package, default: () => ({ cpu: 2, memory: 1024 * 2, disk: 100 }) },
+  minimum: { type: Object as Package, default: () => ({ cpu: 1, memory: 1024 * 2, ssd: 15 }) },
+  standard: { type: Object as Package, default: () => ({ cpu: 2, memory: 1024 * 2, ssd: 100 }) },
```

Comment:
```suggestion
  minimum: { type: Object as Package, default: () => ({ cpu: 1, memory: 1024 * 2, disk: 15 }) },
  standard: { type: Object as Package, default: () => ({ cpu: 2, memory: 1024 * 2, disk: 100 }) },
```

Why did we update `disk` to `ssd` here?

Reply:
It was thabet's request. @xmonader

Reply:
it can be internally defined as disk, but to the user it should be shown as SSD Storage or SSD Disk